processing/src/main/java/io/druid/query/aggregation/AggregatorFactory.java

```diff
@@ -63,6 +59,20 @@
   public abstract Object combine(Object lhs, Object rhs);
 
   /**
+   * Creates an AggregateCombiner to combine rollup aggregation results from serveral "rows" of different indexes during
+   * index merging. AggregateCombiner implements the same logic as {@link #combine}, with the difference that it uses
+   * {@link io.druid.segment.ColumnValueSelector} and it's subinterfaces to get inputs and implements {@code
+   * ColumnValueSelector} to provide output.
+   *
+   * @see AggregateCombiner
+   * @see io.druid.segment.IndexMerger
+   */
+  public AggregateCombiner makeAggregateCombiner()
+  {
+    throw new UOE("[%s] does not implement makeAggregateCombiner()", this.getClass().getName());
```

Comment:
Updated comment. BTW nothing is broken right after this PR, because it just adds functionality, but doesn't use it yet. Actual breakage will happen after N more PRs. However they all target for Druid 0.11.